python/sglang/srt/managers/tp_worker_overlap_thread.py

```diff
@@ -75,7 +75,7 @@ def __init__(
         # Launch threads
         self.input_queue = Queue()
         self.output_queue = Queue()
-        self.forward_stream = torch.get_device_module(self.device).Stream()
+        self.forward_stream = torch.get_device_module(self.worker.device).Stream()
```

Comment:
Same: dose hpu platform use cpu as device_type for dispatching? Or, dose not have any concept like stream? 

Further more: How can we distinguish when to use `hpu` and `cpu`? According to this PR, I found it seems to be an unify memory platform, and I want a confirmation XD

Reply:
hpu has higher host overhead, so my plan is to run the scheduler thread and move all input tensors to hpu in the tp_worker forward thread

Reply:
> hpu has higher host overhead, so my plan is to run the scheduler thread and move all input tensors to hpu in the tp_worker forward thread

I see. It is better to make a `use_pure_cpu_scheduler() -> bool` function in `srt/utils.py`, and make a `maybe_move_inputs_to_device` in tp_worker